libs/category/src/drivers/magento/transformers/category-page-config-transformer.service.ts

```diff
@@ -25,15 +26,16 @@ export class DaffMagentoCategoryPageConfigTransformerService {
   }
 
   private transformAggregate(filter: MagentoAggregation): DaffCategoryFilter {
+		const filterType = this.transformAggregateType(filter.type)
     return {
       label: filter.label,
-      type: this.transformAggregateType(filter.type),
+      type: filterType,
 			name: filter.attribute_code,
 			items_count: filter.count,
 			options: filter.options.map(option => {
 				return {
 					label: option.label,
-					value: option.value,
+					value: filterType !== DaffCategoryFilterType.Range ? option.value : this.transformRangeValue(option.value),
```

Comment:
I think `filterType === DaffCategoryFilterType.Range ? this.transformRangeValue(option.value) : option.value` is a little easier to read and understand.